DigitalLearningSolutions.Data/DataServices/UserDataService/UserDataService.cs

```diff
@@ -86,6 +86,8 @@ string emailAddress
 
         int GetDelegateCountWithAnswerForPrompt(int centreId, int promptNumber);
         void DeleteAllAnswersForPrompt(int centreId, int promptNumber);
+
+        void DeactivateDelegateUser(int delegateId);

```

Comment:
Can you move this up next to the other methods in the DelegateUserDataService file?